### ISSUE TITLE
Deep Classification, Embedding & Text Generation (E3) - Ettinger et al 2018

### ISSUE THREAD

**Issue by HyunkuKwon** [Computational-Content-Analysis-2020/Readings-Responses-Spring#35]
Post questions about the following exemplary reading here:

Ettinger, A., Elgohary, A., Phillips, C., Resnik, P. (2018). “[Assessing Composition in Sentence Vector Representations](https://www.aclweb.org/anthology/C18-1152.pdf).” Proceedings of the 27th International Conference on Computational Linguistics.

**Comment by wanitchayap** [comment]
The paper attributed BOW's higher performance in Content2Probe than the other complexed models to the sacrifice of content to other information in complexed models. Does this mean adding more parameters/nodes/layers to these complexed models will necessary yield better performance? However, like the paper Primer to BERTology mentioned about redundancy in parameters' information, how can we balance the tradeoff?

**Comment by timqzhang** [comment]
There is a potential concern that the paper tends to overcorrect the sentences to get the grammar be clean. As mentioned in pp.1795, Table 1, the sentences in the table are with no doubt grammatically correct, but too formal, which seems unrealistic to human to speak out. I understand that such a strict criteria could wipe out all the potential incorrect sentences, but it could also mis-wipe out the correct sentences with flexible grammar patterns. How to tackle this case?

**Comment by Yilun0221** [comment]
I really liked the framework that the researchers used to build the event representations of sentences. This part perfectly combines semantic structure and computation methods. My question focuses on the deleted sentences. In this research, the researchers deleted grammatically incorrect sentences. However, in the real life, it is very common that what people say in daily life does not fully comply with the rules of grammar. I wonder whether this action would cause any bias?

**Comment by linghui-wu** [comment]
I agree with what pointed out by @timqzhang and @Yilun0221 that the strive for grammatical correctness might be a potential source of bias. So I hope to know why they adopted such a pre-processing strategy?

**Comment by harryx113** [comment]
I wonder why this paper did not tune any hyperparameter and why not considering adding a few hidden layers or using RNN models?

**Comment by tianyueniu** [comment]
It was mentioned in the article that 'no model shows any substantial ability to capture semantic role systematically', I wonder why is that the case, given that generated text has met strict semantic & lexical constraint?

**Comment by WMhYang** [comment]
This paper sheds light on how to distangle sentence composition, which is a fresh angle to better embed human languages. However, as @timqzhang mentioned, the sentences generated are weird since the algorithm only cares about "correct" rather than "reasonable". (I remember that when I was in high school, I wrote such a sentence in my essay for my English class, and my teacher gave me a comment: "No one talked like you. Please do not show off your clauses".) As a result, I wonder is it possible that we also delete these weird sentences?

**Comment by minminfly68** [comment]
This paper is interesting to read and pays some focus on the extraction of meaning. Also, this paper intends to let us understand some difficulties in NLP that it might be difficult to capture the human-like connections. I am wondering how could the way author proposed to overcome the difficulty have real-time influences and how we might find approaches to understand the connections by deductions?

**Comment by bazirou** [comment]
My general question is whether we can extend the basic model to other research areas as the author utilized approaches from Psychology and Neuro Science.

**Comment by timhannifan** [comment]
One limitation of the method used in this paper, compared to other ML approaches we've seen, is the limit imposed on input vectors. The authors extended the size of this space through a generative process, but the methodology is still limited by the words that appear in training. This seems to compromise the ability to generalize to unseen text.
